components/SearchForm.js

```diff
@@ -41,7 +41,7 @@ const SearchButton = styled(Flex)`
 class SearchForm extends React.Component {
   handleSubmit = event => {
     const searchInput = event.target.elements.q;
-    this.props.router.push({ pathname: '/search', query: { q: searchInput.value } });
+    this.props.router.push({ pathname: '/search', query: { q: searchInput.value, sortBy: 'ACTIVITY' } });
```

Comment:
Is that really needed, can't we have this the default another way?

Reply:
Only reason I added this was to add `sortBy="ACTIVITY"` to the query string in the url; but thinking further I think this isn't needed as whether we add that or not the sort by is defaulted to `ACTIVITY` anyways. Therefore I've removed this part.